.github/workflows/run-mypy-on-modified-cdk-files.yml

```diff
@@ -22,7 +23,7 @@ jobs:
         id: changed-py-files
         uses: tj-actions/changed-files@v23
         with:
-          files: "airbyte-cdk/python/airbyte_cdk/**/*.py"
+          files: "**/*.py"
```

Comment:
I think this is fine, but this will enforce mypy on all changed python files — what are some areas that might be surprised with this?